### PR TITLE
Added image substitutions

### PR DESCRIPTION
**Pull request by thegalah**

### CODE REVIEW

File: src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py [hunk 1]
```diff
@@ -1032,6 +1046,16 @@ def _install_k8_secret(acr, dns_prefix, client_id, client_secret,
     except FileNotFoundError as error:
         raise CLIError(error)
 
+def _map_substitutions(yaml, substitutions):
+    """
+    Returns a mapped string containing all the substitutions
+    provided in the subsitutions object
+    """
+    for key in substitutions:
+        print(key)
```
Comment (peterj) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py]: probably no need to print it out

File: src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py [hunk 2]
```diff
@@ -1010,8 +1010,18 @@ def _install_k8_secret(acr, dns_prefix, client_id, client_secret,
...
+
+        substitutions = {
+            '$TENX_PRIVATE_REGISTRY$' : acr,
+            '$TENX_STORAGE_ACCOUNT$' : config_storage,
+            '$TENXBUILDER_IMAGE$' : 'mindaro/tenx-build-service',
+            '$TENXCONFIG_IMAGE$' : 'mindaro/tenx-config-service',
+            '$TENXEXEC_IMAGE$' : 'mindaro/tenx-execute-service',
+            '$TENXRSRC_IMAGE$' : 'mindaro/tenx-rsrc-service',
+            '$TENX_NGINX_IMAGE' : 'mindaro/tenx-nginx-gateway',
+            '$TENXCLIENT_IMAGE' : 'xinyan/tenxclient'
```
Comment (peterj) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py]: there's not image for tenxclient under mindaro repository?

File: src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py (hunk 1 above)
Comment (saurabsa) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py]: >print(key) [](start = 8, length = 10)

Please remove this.

File: src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py [hunk 3]
```diff
@@ -1032,6 +1046,16 @@ def _install_k8_secret(acr, dns_prefix, client_id, client_secret,
     except FileNotFoundError as error:
         raise CLIError(error)
 
+def _map_substitutions(yaml, substitutions):
```
Comment (saurabsa) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py]: >yaml [](start = 23, length = 4)

Rename it to something general, may be 'content'.

File: src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py (hunk 2 above)
Comment (thegalah) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py]: There isn't one at the moment (there were no source files to create the CI with).  I can push a copy to the Mindaro repository though

Comment (sanar-microsoft) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py]: $TENX_NGINX_IMAGE$
$TENXCLIENT_IMAGE$

**Pull request merged**